server/config.ini

```diff
@@ -6,44 +6,26 @@ address =
 
 [keymap_player1]
 
-iidx_1 = m
-iidx_2 = k
-iidx_3 = ,
-iidx_4 = l
-iidx_5 = .
-iidx_6 = ;
-iidx_7 = /
-
-turntable_cw = [
-turntable_ccw = ]
-
-popn_1 = z
-popn_2 = s
-popn_3 = x
-popn_4 = d
-popn_5 = c
-popn_6 = f
-popn_7 = v
-popn_8 = g
-popn_9 = b
-
-jubeat_1 = 4
-jubeat_2 = 5
-jubeat_3 = 6
-jubeat_4 = 7
-jubeat_5 = r
-jubeat_6 = t
-jubeat_7 = y
-jubeat_8 = u
-jubeat_9 = f
-jubeat_10 = g
-jubeat_11 = h
-jubeat_12 = j
-jubeat_13 = v
-jubeat_14 = b
-jubeat_15 = n
-jubeat_16 = m
-
-start = o
-select = p
-
+iidx_1 = z
+iidx_2 = s
+iidx_3 = x
+iidx_4 = d
+iidx_5 = c
+iidx_6 = f
+iidx_7 = v
+
+p1_turntable_cw = [
+p1_turntable_ccw = ]
```

Comment:
[NOTE]
In INIFINITAS, it is set to the Shift key, but the Shift key needs to be converted into a key code with the process `keyboard.Key(shift_l)`, so I changed it to the `[` key etc.